### PR TITLE
Fixed spelling error and clarified text.

### PR DESCRIPTION
**Pull request by jnmorley**
Corrected the spelling of "Running" in the section "Running JupyterLab Notebooks" in "Getting Started with Jupyter," and clarified the text in the first paragraph.

In the text clarification, I used HTML Entities. These render slightly different in Jupyter Lab then they do in the generated HTML.

### CODE REVIEW

**Review by jukent — APPROVED** [review]
This looks great @jnmorley! Thanks for catching that spelling error. I like the arrows to indicate navigating the menu bar. On your read-through, it'd be great if you edited any other places where this could be done for a consistent style.

File: foundations/jupyterlab.ipynb [hunk 1]
```diff
@@ -243,9 +243,9 @@
     "id": "b0Zv8pF8e7Y0"
    },
    "source": [
-    "## Runnning JupyterLab Notebooks\n",
+    "## Running JupyterLab Notebooks\n",
```
Comment (mgrover1) [inline foundations/jupyterlab.ipynb:246]: Good catch!

**Review by mgrover1 — APPROVED** [review]
Thanks for finding this fix @jnmorley

**Comment by brian-rose** [comment]
@jnmorley now that you've got two approvals, you should feel free to hit the green "Squash and merge" button. Congrats on your first contribution!

**Pull request merged**